requirements.txt

```diff
@@ -31,10 +28,18 @@ mypy==0.782
 # Upstream url: https://pypi.python.org/pypi/pytest-mock
 pytest-mock==1.1
 
+# PEP 484
+# License: PSF
+# Upstream url: https://github.com/python/typing
+typing==3.6.4
```

Comment:
do we need to put typing as deps as it should come default in py3?

Reply:
Oh, good question. This was from an older version of the requirements.txt, I'll see if there's any issue if I remove it!

- [ ] remove typing requirement

Reply:
Yep, this seems superfluous. Nice catch!